lib/test.gi

```diff
@@ -124,6 +124,8 @@ InstallGlobalFunction(ParseTestInput, function(str, ignorecomments, fnam)
       Append(inp[Length(inp)], lines[i]{[3..Length(lines[i])]});
       Add(inp[Length(inp)], '\n');
       i := i+1;
+    elif Length(lines[i]) > 1 and lines[i]{[1..2]} = ">\t" then
```

Comment:
Maybe it is just me, but wouldn't this make the code a bit easier to understand?
```suggestion
    elif StartsWith(lines[i], ">\t") then
```